### DISCUSSION TITLE
Fix Git LFS Limitations

### DISCUSSION THREAD

**Discussion by Aristocrat-jt** [git-for-windows/git#3449] [Ideas]
Looks like someone has already fixed this.

https://github.com/xLs/git/commit/54e8a8f28e2560bb57d8ba0bc1e4b73db46b108b

What will it take to merge this back in?

**Comment by rimrul** [comment]
Most code (including comments) in that commit seems to be blatantly ripped fom #2179. All jammed into a single commit with a poor commit message and no attribution to the original authors.

It's basically #2179, but worse in almost every way imaginable (the only good thing about it seems to be that it's based on a newer base commit).

Chances of that ever being merged are pretty much none.

**Reply by PhilipOakley** [reply]
Yes, It would have been useful to have a comment (from that 'fix' 54e8a8f) linked back to https://github.com/git-for-windows/git/pull/2179 - I'd missed that 'suggestion'.

(I wonder if the test issue of using actual large files throughout `t/` was looked at)
